src/client/client.go

```diff
@@ -246,7 +323,8 @@ func DefaultDialOptions() []grpc.DialOption {
 		grpc.WithBlock(),
 
 		// If no connection is established in 30s, fail the call
-		grpc.WithTimeout(30 * time.Second),
+		// grpc.WithTimeout(30 * time.Second),
+		grpc.WithTimeout(5 * time.Second),
```

Comment:
I'm guessing this was just a temporary change? You probably want to either change it back or remove line 326 and update the comment.